exercises/practice/markdown/lib/markdown.ex

```diff
@@ -70,10 +70,9 @@ defmodule Markdown do
   end
 
   defp patch(l) do
-    String.replace_suffix(
-      String.replace(l, "<li>", "<ul>" <> "<li>", global: false),
-      "</li>",
-      "</li>" <> "</ul>"
-    )
+    String.replace(l, "<li>", "<ul>" <> "<li>", global: false)
+    |> String.reverse()
+    |> String.replace(String.reverse("</li>"), String.reverse("</li></ul>"), global: false)
+    |> String.reverse()
```

Comment:
@neenjaw What do you think about this change to the "pre-refactor" boilerpalte? It's a bit ridiculous, but it fixes the tests and I think makes it clear it needs to be refactored 😁

Reply:
Ya, it is certainly clear that it could be improved :P

I like it ✅